portfolio.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+$db = new PDO('mysql:host=127.0.0.1; dbname=osamasCMSDB', 'root');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$projectTitleQuery = $db->prepare("SELECT `projectTitle` FROM `portfolio`");
```

Comment:
same as index.php file - do all selects in one query

Reply:
Done.